src/methods.ts

```diff
@@ -0,0 +1,72 @@
+import snapshot from '@snapshot-labs/strategies';
+import redis from './redis';
+import { sha256, rpcSuccess, rpcError, getBlockNum } from './utils';
+
+interface GetVpRequestParams {
+  address: string;
+  network: string;
+  strategies: any[];
+  snapshot: number | 'latest';
+  space: string;
+  delegation?: boolean;
```

Comment:
This is causing an error

```
Argument of type 'boolean | undefined' is not assignable to parameter of type 'boolean'.
```
Should i change it to `boolean | undefined` in the strategies

Reply:
@ChaituVR This will be fixed in https://github.com/snapshot-labs/snapshot-strategies/pull/891